content/en/docs/tasks/manage-kubernetes-objects/kustomization.md

```diff
@@ -835,8 +853,7 @@ Kustomize has the concepts of **bases** and **overlays**. A **base** is a direct
 set of resources and associated customization. A base could be either a local directory or a directory from a remote repo,
 as long as a `kustomization.yaml` is present inside. An **overlay** is a directory with a `kustomization.yaml` that refers to other
 kustomization directories as its `bases`. A **base** has no knowledge of an overlay and can be used in multiple overlays.
-An overlay may have multiple bases and it composes all resources
-from bases and may also have customization on top of them.
+An overlay may have multiple bases and it composes all resources from bases and may also have customization on top of them.
```

Comment:
This sentence is a little awkward, even rephrased. Is it possible to provide more clarity here?

```suggestion
An overlay may have multiple bases, and composes all resources from bases. It may also have customizations on top of them.
```